src/main/webapp/src/app/location/location.component.html

```diff
@@ -1,8 +1,13 @@
 <div id="location-container">
     <h1>Welcome to Hidden Gems!</h1>
     <h4>To use Hidden Gems, please allow location services. If not possible select a default location [Sydney, Adelaide].</h4>
-    <button id="geolocationButton" (click)="enableNavigatorLocation()" type="button"><i></i>Find My Location</button>
-    <button id="sydneyButton" (click)="getSydneyLocation()" type="button"><i></i>Sydney</button>
-    <button id="adelaideButton" (click)="getAdelaideLocation()" type="button"><i></i>Adelaide</button>
+  
+    <button mat-raised-button color="primary" id="geolocationButton" (click)="enableNavigatorLocation()" type="button"><i></i>Find My Location</button>
+    <button mat-button [matMenuTriggerFor]="menu">Select your city</button>
```

Comment:
nit: maybe add a little down arrow symbol at the end of this to make it clear it's a dropdown menu?

Reply:
added